cluster_management/src/main/java/com/pinterest/rocksplicator/Utils.java

```diff
@@ -281,6 +281,22 @@ public static void backupDB(String host, int adminPort, String dbName, String hd
     }
   }
 
+  public static void backupDBWithLimit(String host, int adminPort, String dbName, String hdfsPath,
+                                       int limitMbs)
+      throws RuntimeException {
+    LOG.error("Backup " + dbName + " from " + host + " to " + hdfsPath);
+    try {
+      Admin.Client client = getAdminClient(host, adminPort);
+
+      BackupDBRequest req = new BackupDBRequest(dbName, hdfsPath);
```

Comment:
are we backing up to hdfs?

Reply:
As current impl, I will still keep it as hdfs. Once we have task framework ready, we can easily switch to S3 or, add both hdfs and s3 support.